{"name": "test_unsuccessful_login[chrome-232@3746-sdjfh]", "status": "passed", "attachments": [{"name": "log", "source": "9dbe1f0f-6698-4244-aaf5-51567fa6c135-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "get_browser", "value": "'chrome'"}, {"name": "username", "value": "'232@3746'"}, {"name": "password", "value": "'sdjfh'"}], "start": 1622322040516, "stop": 1622322040926, "uuid": "2f660f06-cd82-4da2-86ce-23211a86e120", "historyId": "0757c8f6dc643b40392c4e89d9f1bf1f", "testCaseId": "1ce5752f9bf5ac0dd16171f6da2bde9d", "fullName": "test_cases.test_user_sign_in.Test_UserSignIn#test_unsuccessful_login", "labels": [{"name": "story", "value": "Cloud Login"}, {"name": "severity", "value": "normal"}, {"name": "tag", "value": "@pytest.mark.usefixtures('add_logs_on_failure', 'get_browser')"}, {"name": "parentSuite", "value": "test_cases"}, {"name": "suite", "value": "test_user_sign_in"}, {"name": "subSuite", "value": "Test_UserSignIn"}, {"name": "host", "value": "DESKTOP-57RCRK3"}, {"name": "thread", "value": "3784-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.test_user_sign_in"}]}